sysrib/server.go

```diff
@@ -126,6 +122,17 @@ func (d *dplane) ProgramRoute(r *ResolvedRoute) error {
 	return err
 }
 
+// deprogramRoute de-programs the route in the dataplane, returning an error on failure.
+func (d *dplane) deprogramRoute(r *ResolvedRoute) error {
```

Comment:
this should accept a context

Reply:
Addressed in next PR: https://github.com/openconfig/lemming/pull/301